stdlib/SparseArrays/test/sparse.jl

```diff
@@ -93,10 +93,6 @@ do33 = fill(1.,3)
     end
 end
 
-@testset "Issue #30006" begin
```

Comment:
Why are you removing this?

Reply:
that was not my intention, sorry - (I did a git fetch upstream before I created the new branch - will be re-inserted.